integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00000_create_table.sql

```diff
@@ -1,78 +1,78 @@
-CREATE SCHEMA gt_iceberg.gt_db2;
```

Comment:
Why do we remove the catalog prefix?

Reply:
I need to reuse this test for the Iceberg catalog with the JDBC back and Hive catalog, So the catalog name should removed

Reply:
OK